poker-pals/src/images/ImageFiles.js

```diff
@@ -0,0 +1,103 @@
+module.exports =  {
+
+    getImage(name){
+        let image = this.imageFiles.find(image => image.name === name);
+        if(image === undefined){
+            return this.imageFiles.find(image => image.name === 'not_found');
```

Comment:
Can I ask what the purpose of this is?

Reply:
which part? the function?

Reply:
the function provides an easier way for <img src> to access resources

Reply:
The purpose of an image that displays when you can't find the image you want.

Reply:
oh, i was trying to figure out a weird behaviour, the app was showing that an image was found but then it rendered as if the image was not there (i figured it out), I'll remove it now